app/assets/stylesheets/application.css

```diff
@@ -4,12 +4,6 @@
  *
  * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
  * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
  *= require_tree .
  *= require_self
  */
```

Comment:
- [ ] The below tests are failing, Please make sure all tests pass without any errors.

![image](https://github.com/Bleu-Yves-Sopoude/manago/assets/83803414/45289376-e445-4f9b-b138-a11349e84e03)

Reply:
![test](https://github.com/Bleu-Yves-Sopoude/manago/assets/101183454/40afd080-4e26-4d34-b1c9-3ee6883390b4)